src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -466,42 +464,34 @@ class SVTResourceResolver: AzSKRoot {
                         }
                     }
 
-                    if($this.IsPartialScanActive -and $this.nonScannedResources.Count -ne 0){
-                        #$this.PublishCustomMessage("Resuming scan from last commit")
-                        if ($this.BuildNames.Count -gt 0 -and ($this.ResourceTypeName -in ([ResourceTypeName]::Build, [ResourceTypeName]::All, [ResourceTypeName]::Build_Release, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User_CommonSVTResources))) {
-                            $this.createBuildSVTResource($projectName,$organizationId,$projectId);
-                        }
-                        if ($this.ServiceConnections.Count -gt 0 -and ($this.ResourceTypeName -in ([ResourceTypeName]::ServiceConnection, [ResourceTypeName]::All, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User_CommonSVTResources))){
-                            $this.createServiceConnectionSVTResource($projectName,$organizationId,$projectId,$projectData)
+                    if($this.IsPartialScanActive -and $this.nonScannedResources.Count -ne 0 -and $this.isDnrrAllProject -eq $false){
```

Comment:
same as above

Reply:
When the loop starts for the next project, we need not load the releases again for any of the projects, all releases have been added to svt in first pass itself